prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -258,5 +268,72 @@ public async Task<ActionResult> FinalizeEnrolleePaperSubmission(int enrolleeId)
 
             return Ok();
         }
+
+        // HEAD: api/Enrollees/potential-paper-enrollee/5
+        /// <summary>
+        /// Gets all paper enrollees and checks whether or not there is a match in dateOfBirth with the current logged on enrollee.
+        /// </summary>
+        [HttpHead("potential-paper-enrollee/{dateOfBirth}")]
+        [Authorize(Roles = Roles.TriageEnrollee + "," + Roles.PrimeEnrollee)]
```

Comment:
Whenever you make an endpoint available to PrimeEnrollees, you have to check their UserId against the record's UserId else you are letting any use looks at / edit any other user's data.